crates/bevy_ptr/README.md

```diff
@@ -1,4 +1,9 @@
-# `bevy_ptr`
+# Bevy Ptr
+
+[![MIT/Apache 2.0](https://img.shields.io/badge/license-MIT%2FApache-blue.svg)](https://github.com/bevyengine/bevy#license)
+[![Crates.io](https://img.shields.io/crates/v/bevy_ptr.svg)](https://crates.io/crates/bevy_ptr)
+[![Crates.io](https://img.shields.io/crates/d/bevy_ptr.svg)](https://crates.io/crates/bevy_ptr)
```

Comment:
Why is crates.io duplicated twice here? Should these be the docs?

Reply:
I will change the text to Downloads and also add the Docs badge